package.json

```diff
@@ -5,16 +5,39 @@
   "description": "React Native in-app purchases and subscriptions made easy.",
   "main": "index.js",
   "types": "index.d.ts",
+  "source": "src/index",
+  "files": [
+    "build.js",
+    "index.js",
+    "index.d.ts",
+    "android",
+    "!android/gradlew*",
+    "!android/.gradle",
+    "!android/gradle",
+    "!android/build",
+    "!android/local.properties",
+    "ios",
+    "!ios/build",
+    "!ios/RNPurchases.xcodeproj/project.xcworkspace",
+    "!ios/RNPurchases.xcodeproj/xcuserdata",
+    "RNPurchases.podspec",
+    "scripts",
+    "!**/__tests__",
+    "!**/.idea",
+    "!**/*.iml",
+    "!**/.DS_Store",
+    "!**/.gitignore"
+  ],
```

Comment:
I ran `npm publish` before adding all this, and it was trying to publish 13.5 MB. After these changes:

```
npm notice 📦  react-native-purchases@3.3.2
npm notice === Tarball Contents ===
npm notice 2.5kB  package.json
npm notice 616B   build.js
npm notice 8.8kB  CHANGELOG.md
npm notice 32.6kB index.d.ts
npm notice 27.7kB index.js
npm notice 1.1kB  LICENSE
npm notice 4.5kB  README.md
npm notice 773B   RNPurchases.podspec
npm notice 3.8kB  android/build.gradle
npm notice 150B   android/gradle.properties
npm notice 0      android/settings.gradle
npm notice 137B   android/src/main/AndroidManifest.xml
npm notice 4.1kB  android/src/main/java/com/revenuecat/purchases/react/RNPurchasesConverters.kt
npm notice 9.9kB  android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java
npm notice 963B   android/src/main/java/com/revenuecat/purchases/react/RNPurchasesPackage.java
npm notice 360B   ios/RNPurchases.h
npm notice 10.6kB ios/RNPurchases.m
npm notice 9.8kB  ios/RNPurchases.xcodeproj/project.pbxproj
npm notice 2.6kB  ios/strip-frameworks.sh
npm notice 1.3kB  scripts/download-purchases-common.sh
npm notice 1.2kB  scripts/download-purchases-framework.sh
npm notice === Tarball Details ===
npm notice name:          react-native-purchases
npm notice version:       3.3.2
npm notice package size:  26.7 kB
npm notice unpacked size: 123.4 kB
```